parent/pom.xml

```diff
@@ -207,14 +218,40 @@
                             <exclude>**/pom.xml</exclude>
                         </excludes>
                         <group>adobe/cq60</group>
+                        <validatorsSettings>
+                            <jackrabbit-filter>
+                                <options>
+                                    <validRoots>/,/libs,/apps,/apps/core/wcm,/etc,/var,/tmp,/content</validRoots>
+                                </options>
+                            </jackrabbit-filter>
+                            <netcentric-aem-classification>
+                                <!-- <disabled>true</disabled>  -->
+                                <defaultSeverity>WARN</defaultSeverity>
```

Comment:
Any reason to set this to `WARN` instead of `ERROR`?

Reply:
yes, you use some deprecated resource type currently, and I cannot easily resolve that!